src/sdltiles.cpp

```diff
@@ -1200,7 +1201,7 @@ static bool draw_window( Font_Ptr &font, const catacurses::window &w, const poin
                 // utf8_width() may return a negative width
                 continue;
             }
-            bool use_draw_ascii_lines_routine = get_option<bool>( "USE_DRAW_ASCII_LINES_ROUTINE" );
+            bool use_draw_ascii_lines_routine = option_use_draw_ascii_lines_routine;
```

Comment:
Do you really need a second variable here? Maybe you can just move declaration ans assignment of `use_draw_ascii_lines_routine` to the top?

Reply:
Thanks for the review. Yes, that would have been nice, but this variable is modified in the `case`-statement below, depending on the cell contents, see line 1240-1243.